common/combat_tactics/wc_combat_tactics.txt

```diff
@@ -0,0 +1,1256 @@
+            flank_has_leader = yes
+			leader = {
+				martial = 12
+			}
+			AND = {
+				OR = {
+					heavy_troops = {
+						who = heavy_infantry
+						value = 0.1
+					}
+					heavy_troops = {
+						who = undead
+						value = 0.1
+					}
+					heavy_troops = {
+						who = knights
+						value = 0.1
+					}
+					heavy_troops = {
+						who = thralls
+						value = 0.1
+					}
+				}
+				OR = {
+					light_troops = {
+						who = light_infantry
+						value = 0.1
+					}
+					light_troops = {
+						who = archmage
+						value = 0.1
+					}
+				}
+			}
+		}
+		modifier = {
+			factor = 3
+            flank_has_leader = yes
+			leader = {
+				martial = 16
+			}
+			AND = {
+				OR = {
+					heavy_troops = {
+						who = heavy_infantry
+						value = 0.15
+					}
+					heavy_troops = {
+						who = undead
+						value = 0.15
+					}
+					heavy_troops = {
+						who = knights
+						value = 0.15
+					}
+					heavy_troops = {
+						who = thralls
+						value = 0.15
+					}
+				}
+				OR = {
+					light_troops = {
+						who = light_infantry
+						value = 0.15
+					}
+					light_troops = {
+						who = archmage
+						value = 0.15
+					}
+				}
+			}
+		}
+		modifier = {
+			factor = 5
+            flank_has_leader = yes
+			leader = {
+				martial = 8
+				is_magic_leader = yes
+			}
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 2 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 4 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 6 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 8 } } }
+		}
+	}
+
+	heavy_infantry_defensive = 1.2
+	undead_defensive = 1.2
+	thralls_defensive = 1.2
+	knights_defensive = 1.2
+	heavy_infantry_offensive = 0.5
+	undead_offensive = 0.5
+	thralls_offensive = 0.5
+	knights_offensive = 0.5
+	light_infantry_defensive = 2
+	archmage_defensive = 2
+	enemy = {
+		group = charge
+		factor = 2
+	}
+}
+
+wc_overwhelm_tactic = {
+	days = 18
+	sprite = 10
+	group = charge
+
+	trigger = {
+		phase = melee
+		OR = {
+			swarm = 0.2
+			animals = 0.2
+			undead = 0.2
+			thralls = 0.2
+		}
+	}
+
+	mean_time_to_happen = {
+		days = 3
+		modifier = {
+			factor = 3
+            flank_has_leader = yes
+			leader = {
+				martial = 12
+			}
+			OR = {
+				heavy_troops = {
+					who = undead
+					value = 0.60
+				}
+				heavy_troops = {
+					who = swarm
+					value = 0.60
+				}
+				heavy_troops = {
+					who = thralls
+					value = 0.60
+				}
+				light_troops = {
+					who = animals
+					value = 0.60
+				}
+			}
+		}
+	}
+
+	thralls_offensive = 0.50
+	undead_offensive = 1
+	swarm_offensive = 1
+	animals_offensive = 1
+	enemy = {
+		group = advance
+		factor = 2
+	}
+}
+
+wc_overrun_tactic = {
```

Comment:
There's already swarm tactics. Maybe... you know, get rid of it and rework the vanilla ones?